amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/preview/PreviewActivityService.java

```diff
@@ -292,5 +308,108 @@ private Double convertProjectCostAmount(AmpFundingAmount projectCost, String cur
             }
         }
     }
+    
+    public List<PreviewWorkspace> getWorkspaces(Long activityId) {
+        List<PreviewWorkspace> previewWorkspaces = new ArrayList<>();
+    
+        try {
+            AmpActivityVersion activity = ActivityUtil.loadActivity(activityId);
+            AmpTeam ownerTeam = activity.getTeam();
+            String ownerInfo = TranslatorWorker.translateText("Workspace where the activity was created");
+            previewWorkspaces.add(new PreviewWorkspace(ownerTeam.getName(), PreviewWorkspace.Type.TEAM, ownerInfo));
+    
+            AmpTeam parentTeam = ownerTeam.getParentTeamId();
+            String path = ownerTeam.getName();
+            
+            while (parentTeam != null) {
+                path = String.format("%s -> %s", path, parentTeam.getName());
+                previewWorkspaces.add(new PreviewWorkspace(parentTeam.getName(),
+                        PreviewWorkspace.Type.MANAGEMENT, path));
+                
+                parentTeam = parentTeam.getParentTeamId();
+            }
+    
+            List<AmpTeam> computedTeams = TeamUtil.getAllTeams().stream()
```

Comment:
Here we are excluding management ws. Lets imagine this functionality
we have a computed ws (computed 1) with computed orgs that includes activity A. We have a management ws (Management 1)that has computed 1 as child. It won't be shown.

Actually now if you have this config and log in to Management 1 the computed activities that are seen in computed 1are not see, I have to check with Octavian since I think this is a bug